src/translib/intf_cmn.go

```diff
@@ -933,6 +936,9 @@ func (app *IntfApp) processGetSpecificIntf(dbs [db.MaxDB]*db.DB, targetUriPath *
 			}
 
 			ifInfo := intfObj.Interface[ifKey]
+			if *app.ygotTarget != ifInfo || *app.ygotTarget != ifInfo.Config || *app.ygotTarget != ifInfo.State {
+				return GetResponse{Payload: payload}, errors.New("Requested get type not supported!")
+			}
```

Comment:
This change might cause issue for following Get request targets: 

- OpenconfigInterfaces_Interfaces_Interface_Aggregation_State
- OpenconfigInterfaces_Interfaces_Interface_Ethernet_SwitchedVlan_State

Can you please check and confirm?

Reply:
Yes, you are right. Could you add those cases in port-channel pull-request?

Reply:
Probably, we need to add those cases in a map and handle it. That would be better handling.